exist-core/src/main/java/org/exist/xquery/functions/math/OneParamFunctions.java

```diff
@@ -145,42 +145,20 @@ public Sequence eval(Sequence[] args, Sequence contextSequence) throws XPathExce
             double calcValue = 0;
             final String functionName = getSignature().getName().getLocalPart();
 
-            if (ACOS.equals(functionName)) {
-                calcValue = Math.acos(value.getDouble());
-
-            } else if (ASIN.equals(functionName)) {
-                calcValue = Math.asin(value.getDouble());
-
-            } else if (ATAN.equals(functionName)) {
-                calcValue = Math.atan(value.getDouble());
-
-            } else if (COS.equals(functionName)) {
-                calcValue = Math.cos(value.getDouble());
-
-            } else if (EXP.equals(functionName)) {
-                calcValue = Math.exp(value.getDouble());
-
-            } else if (EXP10.equals(functionName)) {
-                calcValue = Math.pow(10.0d, value.getDouble());
-
-            } else if (LOG.equals(functionName)) {
-                calcValue = Math.log(value.getDouble());
-
-            } else if (LOG10.equals(functionName)) {
-                calcValue = Math.log10(value.getDouble());
-
-            } else if (SIN.equals(functionName)) {
-                calcValue = Math.sin(value.getDouble());
-
-            } else if (SQRT.equals(functionName)) {
-                calcValue = Math.sqrt(value.getDouble());
-
-            } else if (TAN.equals(functionName)) {
-                calcValue = Math.tan(value.getDouble());
-
-            } else {
-                throw new XPathException(this, ERROR, "Function " + functionName + " not found.");
-            }
+            calcValue = switch (functionName) {
```

Comment:
can we make this a `final calcValue = switch (functionName) { … default -> 0; };`?

Reply:
nice catch!